src/app/capture_thread.cpp

```diff
@@ -36,9 +36,11 @@ CaptureThread::CaptureThread(int cam_id)
   control->addChild( (VarType*) (captureModule= new VarStringEnum("Capture Module","DC 1394")));
   captureModule->addFlags(VARTYPE_FLAG_NOLOAD_ENUM_CHILDREN);
   captureModule->addItem("DC 1394");
+  captureModule->addItem("V4L (USB)");
```

Comment:
Please call this "Video 4 Linux", to be precise. PtGrey and other manufacturers have their own custom USB interface.